### DISCUSSION TITLE
Add a connection for Oracle Cloud Fusion ERP BI Publisher via REST API that can execute SQL

### DISCUSSION THREAD

**Discussion by gd78457** [dbeaver/dbeaver#21378] [Ideas]
Oracle Cloud Fusion ERP does not give any direct access to the backend Oracle Databases that run the Oracle Cloud Fusion ERP SaaS application. However there is a way to create a REST API to connect via the BI Publisher that can allow you to submit SQL and have results returned in XML format and I had the idea that if DBeaver could be used to connect to that REST API then the DBeaver UI would be so helpful to interacte with the REST API. 

Is there a similar use case already released in DBeaver that would be the starting point for the enhancement?

Do we think that an enhancement like this would be even possible in DBeaver?

**Comment by E1izabeth** [comment]
It's not possible now and as far as i know we haven't received any similar requests earlier. I believe we should think about it. Could you create an issue about working with Oracle Cloud Fusion ERP via DBeaver?

**Comment by krokozyab** [comment]
https://github.com/krokozyab/ofjdbc
